test/integration/test-http-response.js

```diff
@@ -51,22 +51,14 @@ http.request(options, function(response) {
 }).end();
 
 // prepare form-receiving http server
-server = http.createServer(function(req, res) {
+var incomingForm = new IncomingForm({uploadDir: common.dir.tmp});
 
```

Comment:
And this one, we need new instance on each request, so it could go into `http.createServer` of  `common.js`.